.coveragerc

```diff
@@ -0,0 +1,15 @@
+[run]
+omit =
+    # These are not covered by any test because it is an experimental API
```

Comment:
Should `benchmarks/` and `examples/` also be included here? Those directories also have Python files in them

Reply:
It's only checking coverage from the root/modin. Those directories are in root so they aren't tested.